Nimble_Snapshots/CurrentTestCaseTracker.swift

```diff
@@ -17,12 +17,8 @@ import XCTest
 
 extension XCTestCase {
     var sanitizedName: String? {
-        guard let fullName = self.name else {
-            return nil
-        }
-
         let characterSet = CharacterSet(charactersIn: "[]+-")
-        let name = fullName.components(separatedBy: characterSet).joined()
+        let name = self.name.components(separatedBy: characterSet).joined()
```

Comment:
Pretty sure is an optional, you can also see it in #133 that has been merge it today. With beta 6 the compiler has to be complaining. Maybe just clean derived data + project and try again.